experimental/cloudstrapper/playbooks/roles/control/tasks/local-dirs.yaml

```diff
@@ -0,0 +1,15 @@
+---
+#backing up certs in bootstrap to pass on to AGW
+
+- name: create certs folder on bootstrapper
+  file: 
+    path: "{{ dirSecretsLocal }}"
+    state: directory
+    mode: '0755'  
+
+    #TODO: Is this really needed?
+- name: install boto3 in local directory
```

Comment:
I see the TODO, perhaps we can clean this up. I see us installing boto3 in aws prereqs as well

Reply:
This file is from an earlier draft. Removed.